Week1/work.py

```diff
@@ -12,7 +12,15 @@ def demo(x):
     '''
 
     ## Code Here
-    return None
+def square(num):
+    return num * num
+
+
+number = float(input("please enter any numeric value:"))
+
+sqre = square(number)
+
+print("the square of a given number", sqre)
```

Comment:
This is a common mistake @all of you are doing you don't put the checking or debugging code in the file that is input() and print() functions and please do not change the name o the function as in here. it was demo which was changed into square. hence It would fail.